app/serializers/file-report.js

```diff
@@ -0,0 +1,5 @@
+import DRFSerializer from './drf';
+
+export default DRFSerializer.extend({
+  primaryKey: 'id',
```

Comment:
Is specifying `id` required?

Reply:
when I initially started this task, I referred a file which has same thing so, just copied 😄